src/layer/marker/Icon.Default.js

```diff
@@ -20,41 +22,14 @@ import * as DomUtil from '../../dom/DomUtil';
 export var IconDefault = Icon.extend({
 
 	options: {
-		iconUrl:       'marker-icon.png',
-		iconRetinaUrl: 'marker-icon-2x.png',
-		shadowUrl:     'marker-shadow.png',
+		iconUrl:     emptyImageUrl,
+		shadowUrl:   emptyImageUrl,
```

Comment:
Why not DivIcon?

Reply:
No reason, really. I made this as a quick hack and didn't think about making it a `DivIcon`.